libc/src/__support/block.h

```diff
@@ -332,29 +353,34 @@ Block<OffsetType, kAlign>::init(ByteSpan region) {
     return {};
 
   region = result.value();
-  if (region.size() < BLOCK_OVERHEAD)
+  if (region.size() < 2 * BLOCK_OVERHEAD)
```

Comment:
Out of curiosity, why does this need to be `2 * BLOCK_OVERHEAD`?

Reply:
Added a comment.